CONTRIBUTING.md

```diff
@@ -0,0 +1,25 @@
+# How to Contribute (*Bit Project Style*)
+
```

Comment:
Consider adding a resource page with videos/github docs. Here's also a learning lab from github: https://lab.github.com/githubtraining/introduction-to-github

> I think we need to have a basics to Github under a new markdown file and be like. here is our guide to getting started with github if you are new to it. CONTRIBUTING.md almost never includes a github guide.

Referencing Daniel's comment, I agree that the resources + basic github stuff should be in another md file

Reply:
Yup! Updated with links, good suggestion.